src/simpleMask.js

```diff
@@ -92,6 +105,13 @@ const simpleMask = (containerEl, options = {}) => {
         }
     };
 
+    /* Assume value is masked if it contains at least one special character */
+    const isValueMasked = value => {
+        return value.split('').some(char => 
```

Comment:
[alternative] I think this function could also be re-written as
```
const isValueMasked = value => {
    arrayOfSpecialCharactersInFormat.some(specialChar => value.indexOf(specialChar) > -1);
}
```
I'm not sure if that is slightly more performant but it does eliminate nested `some`s :shrug:

Reply:
Ooh yes, I like this better. Good idea!